.changeset/orange-news-flash.md

```diff
@@ -0,0 +1,17 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Autosuggest
+---
+
+**Autosuggest:** Add `automaticHighlights` prop
+
+Introduces the `automaticHighlights` prop, which uses the input value to automatically highlight either the `matching` or `remaining` portion of each suggestion.
```

Comment:
What are your thoughts on renaming `automaticHighlights` to `suggestionHighlight`?

```ts
<Autosuggest suggestionHighlight="matching" />
```